README.md

```diff
@@ -1,3 +1,39 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+## Versions
+
+Built with Ruby 2.7.2, CSV, Rspec 3.12.0
+
+## Approach to Reach Goals
+
+**We plan to meet daily at 2:30pm MST for stand-ups and work together or be available for most of the afternoon and evening with a dinner break.**
+
+**Our team plans to use _Github Projects_ to organize our tasks/tickets that need to be accomplished in our project. We are also using _Miro_ to organize our thoughts and to mind-map/brainstorm our whole project.  We collectively add inputs when brainstorming and then we can edit tickets in our _Github Projects_.** 
+- [GitHub Project](https://github.com/users/grace-joh/projects/3)
+- [Miro Board](https://miro.com/app/board/uXjVPiopTmI=/)
+
+**To decide on our approach, we looked at a couple ways of organizing our ideas and setting up where we would track everyone's work. We settled on using Miro for planning and brainstorming, and using github projects to organize individual tasks, which we picked over trello because we wanted to try something most of us had not used before.**
+
+**Our team will first brainstorm at a high level to create our code’s functionality. We will observe the data files and methods required in this project to decide how to store our data in classes that are modular in its state and behavior. As we build, we will look for opportunities to refactor our initial classes to include modules, inheritance, or utility classes to maintain single responsibility per class.**
+
+## Group DTR
+
+- [DTR Document](https://docs.google.com/document/d/1Nqv1R2Ognb3WXtzAy6NgxfQZs2xR7j-oHdXkRTyv0K4/edit?usp=sharing)
+Completed on Feb. 27, 2023
+
+## Contributors
+
+Amy Spears
+- [LinkedIn](https://www.linkedin.com/in/amy-marie-spears-900997105/)
+- [GitHub](https://github.com/amspears007)
+Grace Joh
+- [LinkedIn](http://www.linkedin.com/in/graceehjoh)
+- [GitHub](https://github.com/grace-joh)
+Julian Beldotti
+- [LinkedIn](https://www.linkedin.com/in/julian-beldotti-1a427824/)
+- [GitHub](https://github.com/JCBeldo)
+Logan Cole
```

Comment:
The names will each either need a new line or a bullet point, or it will not format properly.

Reply:
```
## Contributors
- Amy Spears
  - [LinkedIn](https://www.linkedin.com/in/amy-marie-spears-900997105/)
  - [GitHub](https://github.com/amspears007)
- Grace Joh
  - [LinkedIn](http://www.linkedin.com/in/graceehjoh)
  - [GitHub](https://github.com/grace-joh)
- Julian Beldotti
  - [LinkedIn](https://www.linkedin.com/in/julian-beldotti-1a427824/)
  - [GitHub](https://github.com/JCBeldo)
- Logan Cole
  - [LinkedIn](https://www.linkedin.com/in/logan-cole-ab2179188/)
  - [GitHub](https://github.com/exasperlnc)
```

Reply:
Okay, it's been updated with 6 eyes.